sdks/python/test-suites/tox/py38/build.gradle

```diff
@@ -112,6 +112,8 @@ task archiveFilesToLint(type: Zip) {
     include "**/*.md"
     include "**/build.gradle"
     exclude "**/node_modules/*"
+    exclude ".github/*"
+    exclude "website/*"
```

Comment:
Did you mean to include this? Why do we need it?

Reply:
Yeah, they're being picked up by the py38 process (not sure why) so I had to manually exclude them from the paths.